src/EnergyPlus/HVACManager.cc

```diff
@@ -1740,14 +1678,14 @@ void SimHVAC(EnergyPlusData &state)
                                 Real64 tempHistOutletNodeDotProd = std::inner_product(
                                     std::begin(ConvergenceHistoryARR), std::end(ConvergenceHistoryARR), std::begin(tempHistOutletNode), 0.0);
 
-                                SlopeTemps =
-                                    (sum_ConvergenceHistoryARR * sum(tempHistOutletNode) -
-                                     double(NumConvergenceHistoryTerms) * tempHistOutletNodeDotProd) /
-                                    (square_sum_ConvergenceHistoryARR - double(NumConvergenceHistoryTerms) * sum_square_ConvergenceHistoryARR);
+                                SlopeTemps = (sum_ConvergenceHistoryARR * sum(tempHistOutletNode) -
```

Comment:
Personal preference: when you have lots of terms with operations and parentheses, I like to create intermediate const variables to store things and make the calculations easier to grok.  I feel confident it doesn't affect the compiler's ability to optimize the evaluation, but can make the code look a lot nicer.  Nothing for now, just commenting.